bot_controller/src/routes.rs

```diff
@@ -173,20 +176,20 @@ pub async fn start_bot(
         .arg("--LadderServer")
         .arg(resolved_game_host)
         .arg("--StartPort")
-        .arg(&game_port)
+        .arg(&game_pass)
         .arg("--OpponentId")
         .arg(opponent_id)
         .current_dir(&bot_path);
 
-    debug!("Starting bot with command {:?}", &command);
-    let mut process = match command.spawn() {
+    info!("Starting bot with command {:?}", &command);
+    match command.spawn() {
         Ok(mut process) => {
             tokio::time::sleep(Duration::from_secs(2)).await;
             match process.try_status() {
                 Ok(None) => {}
                 Ok(Some(exit_status)) => {
                     return Err(ProcessError::StartError(format!(
-                        "Bot {bot_name} has exited within 5 seconds with status {exit_status}"
+                        "Bot {bot_name} has exited within 2 seconds with status {exit_status}"
```

Comment:
Perhaps the number of seconds can be made a variable/constant here, just so this doesn't get out of sync again if we change it in the future.